web-next/src/routes/(root)/notifications.tsx

```
@@ -0,0 +1,50 @@
+import { Show } from "solid-js";
+import {
+  createPreloadedQuery,
+  loadQuery,
+  useRelayEnvironment,
+} from "solid-relay";
+import { NotificationList } from "~/components/NotificationList.tsx";
+import { Title } from "~/components/Title.tsx";
+import { useLingui } from "~/lib/i18n/macro.d.ts";
+import type { notificationsPageQuery } from "./__generated__/notificationsPageQuery.graphql.ts";
+
+const notificationsPageQuery = graphql`
```

Biome (2.1.2)

[error] Shouldn't redeclare 'notificationsPageQuery'. Consider to delete it or rename it.

'notificationsPageQuery' is defined here:


(lint/suspicious/noRedeclare)